notebooks/en/structured_generation_vision_languag_models.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<p>Would also be nice to briefly describe what "Tags" is.</p><p> </p><p>"Now, we are going to create a function that will define...We want to extract Tags for an object in the image..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>